KsApi/models/templates/ProjectStatsTemplate.swift

```diff
@@ -11,6 +11,7 @@ extension Project.Stats {
     goal: 2_000,
     pledged: 1_000,
     staticUsdRate: 1.0,
-    updatesCount: 1
+    updatesCount: 1,
+    usdExchangeRate: 1.56
```

Comment:
Should we test this with a number similar to what we were getting from the API? More trailing decimals is probably a good testing use case.